js/main.js

```diff
@@ -518,7 +518,8 @@ function generateHTMLCalendarDay(square, dateObj, isCurrentMonth, dayObj, curren
         travelAdded = true;
         if (dayObj.travel !== currentTravel.location) {
           currentTravel.location = dayObj.travel;
-          currentTravel.style = 'travel-' + (parseInt(currentTravel.style[7]) + 1);
+          var travelStyleIndex = parseInt(currentTravel.style[7]);
+          currentTravel.style = 'travel-' + (travelStyleIndex + 1);
           if (currentTravel.style[7] === '5') {
```

Comment:
You should use `travelStyleIndex` here, too. If you are using `currentTravel.style[7]` elsewhere, make sure you change that too